### PR TITLE
abnormal-security-source docs

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 1]
```diff
@@ -61,175 +61,181 @@ In this section, we'll introduce the following concepts:
...
+<div className="box smallbox17 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloud-to-cloud-source-versions"><img src={useBaseUrl('img/icons/business/SaaS.png')} alt="icon" width="50"/><h4>Cloud-to-Cloud Versions</h4></a>
   <p>Sources in the Cloud-to-Cloud Integration Framework need updates over time to maintain data collection.</p>
   </div>
 </div>
-<div className="box smallbox17 card">
+<div className="box smallbox18 card">
   <div className="container">
   <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-source"><img src={useBaseUrl('img/integrations/security-threat-detection/crowdstrike.png')} alt="thumbnail icon" width="65"/><h4>CrowdStrike</h4></a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:161]:
```suggestion
```

Comment (jpipkin1) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:161]: This should be removed. It appears in error in the Code42 card.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+<img src={useBaseUrl('img/send-data/abnormal-security-logo.png')} alt="abnormal-security-logo" width="90" />
+
+Abnormal Security is a behavioral AI-based email security platform that learns the behavior of every identity in a cloud email environment and analyzes the risk of every event to block the most sophisticated attacks.
+
+The Abnormal Security integration ingests threat data identified by the abnormal threat log using the [Abnormal Security API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1).
+
+## Data Source
+
+The Abnormal Security source fetches the list of threat IDs using the [Threats API ](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats) and for each threat ID it fetches details of the threat using the [Threat Details API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:18]:
```suggestion
The Abnormal Security source fetches the list of threat IDs using the [Threats API ](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats), and for each threat ID it fetches details of the threat using the [Threat Details API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The Abnormal Security source requires you to provide an authentication token. To obtain these, follow the steps below.
+1. Sign in to the [Abnormal Security](https://portal.abnormalsecurity.com/) platform.<br/><img src={useBaseUrl('img/send-data/abnormal-login.png')} alt="abnormal-login" style={{border: '1px solid black'}} width="400" />
+1. In the **Enhance** section, click on the **Settings** option.<br/><img src={useBaseUrl('img/send-data/abnormal-settings.png')} alt="abnormal-settings" style={{border: '1px solid black'}} width="800" />
+1. From the **Settings** page, copy and save the authentication token.
+
+## States
+
+The Abnormal Security source is a security platform that provides AI-based email security.
+When you create a Abnormal Security source, it goes through the following stages.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:30]:
```suggestion
When you create an Abnormal Security source, it goes through the following stages.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+1. **Authenticated**. The Source successfully authenticated with Abnormal Security.
+1. **Collecting**. The Source is actively collecting data from Abnormal Security.
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+## Create Abnormal Security Source
+
+When you create a Abnormal Security Source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:43]:
```suggestion
When you create an Abnormal Security Source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+To configure a Abnormal Security Source, follow the steps below:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:45]:
```suggestion
To configure an Abnormal Security Source, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+1. Enter a **Name** for the Source. The description is optional. <br/><img src={useBaseUrl('img/send-data/abnormal-config.png')} alt="abnormal-config" style={{border: '1px solid black'}} width="400" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:53]:
```suggestion
   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema, it is ignored, also known as dropped.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This looks fantastic, @JV0812! Very nicely done. I only have a few very minor comments.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md (hunk 1 above)
Comment (JV0812) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:161]: Thanks for pointing out this John. I have updated this.

**Review by jpipkin1 — APPROVED** [review]

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
+---
+id: abnormal-security-source
+title: Abnormal Security Source
+sidebar_label: Abnormal Security
+description: Learn how to collect abnormal threat log from the Abnormal Security.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:5]:
```suggestion
description: Learn how to collect abnormal threat log from the Abnormal Security source.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+<img src={useBaseUrl('img/send-data/abnormal-security-logo.png')} alt="abnormal-security-logo" width="90" />
+
+Abnormal Security is a behavioral AI-based email security platform that learns the behavior of every identity in a cloud email environment and analyzes the risk of every event to block the most sophisticated attacks.
+
+The Abnormal Security integration ingests threat data identified by the abnormal threat log using the [Abnormal Security API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1).
+
+## Data Source
+
+The Abnormal Security source fetches the list of threat IDs using the [Threats API ](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats), and for each threat ID it fetches details of the threat using the [Threat Details API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/abnormal-security-source.md:18]:
```suggestion
The Abnormal Security source fetches the list of threat IDs using the [Threats API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats), and for each threat ID it fetches details of the threat using the [Threat Details API](https://app.swaggerhub.com/apis-docs/abnormal-security/abx/1.4.1#/Threats/get_threats__threatId_).
```

**Pull request merged**